### PR TITLE
New Chinese translation

### PR DESCRIPTION
**Pull request by kalifun**
New Chinese translation documents

### CODE REVIEW

File: language/resource.language.zh_cn/strings.po [hunk 1]
```diff
@@ -0,0 +1,1604 @@
...
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31165"
+msgid "Seekbar behaviour on delayed pause"
+msgstr "延迟暂停时的 Seekbar 行为"
+
+#: /1080i/Includes_Defaults.xml
+msgctxt "#31172"
+msgid "You have watched $INFO[Window(Home).Property(TVShows.Watched),[COLOR=main_fg_90][B],[/B][/COLOR]] of $INFO[Window(Home).Property(TVShows.Count),[COLOR=main_fg_90][B],[/B][/COLOR]] tvshows in your library.[CR]The episode $INFO[Window(Home).Property(LatestEpisode.1.EpisodeTitle),[COLOR=main_fg_90][B],[/B][/COLOR]]$INFO[Window(Home).Property(LatestEpisode.1.ShowTitle), from the show [COLOR=main_fg_90][B],[/B][/COLOR]] was recently added."
+msgstr "您已经观看了$INFO[Window(Home)。$INFO[Window(Home)的财产(TVShows.Watched),[COLOR=main_fg_90][B],[/B][/COLOR]]。财产（TVShows.Count），[COLOR=main_fg_90][B]，[/B][/COLOR]]您图书馆中的电视节目。[CR]剧集$INFO[窗口（主页）。属性（最新剧集.1。剧集标题），[COLOR=main_fg_90][B]，[/B][/COLOR]]$INFO[窗口（主页）。属性（最新剧集.1。ShowTitle），来自节目[COLOR=main_fg_90][B]，[/B][/COLOR]]最近添加了。"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Don't translate the `$INFO[]` tags as these are skin values.

File: language/resource.language.zh_cn/strings.po [hunk 2]
```diff
@@ -0,0 +1,1604 @@
...
+
+msgctxt "#31174"
+msgid "Navigation"
+msgstr "导航"
+
+#: /1080i/Includes_Defaults.xml
+msgctxt "#31175"
+msgid "You have watched $INFO[Window(Home).Property(Movies.Watched),[COLOR=main_fg_90][B],[/B][/COLOR]] of $INFO[Window(Home).Property(Movies.Count),[COLOR=main_fg_90][B],[/B][/COLOR]] movies in your library.[CR]The movie $INFO[Window(Home).Property(LatestMovie.1.Title),[COLOR=main_fg_90][B],[/B][/COLOR]]$INFO[Window(Home).Property(LatestMovie.1.Year), released in [COLOR=main_fg_90][B],[/B][/COLOR]] was recently added."
+msgstr "您已经观看了$INFO[Window(Home)。$INFO[Window(Home)的财产(Movies.Watched),[COLOR=main_fg_90][B],[/B][/COLOR]]。库中的属性（Movies.Count），[COLOR=main_fg_90][B]，[/B][/COLOR]]电影。[CR]电影$INFO[窗口（主页）。属性（最新电影.1。标题），[COLOR=main_fg_90][B]，[/B][/COLOR]]$INFO[窗口（主页）。属性（最新电影.1。年），在[COLOR=main_fg_90][B]发布，[/B][/COLOR]]最近添加了。"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Same here.

File: language/resource.language.zh_cn/strings.po [hunk 3]
```diff
@@ -0,0 +1,1604 @@
...
+msgstr "您有$INFO[窗口（主页）。Property(Movies.Count),[COLOR=main_fg_90][B],[/B][/COLOR]] movies$COMMA $INFO[Window(Home).Property(TVShows.Count),[COLOR=main_fg_90][B],[/B][/COLOR]]电视节目$COMMA $INFO[Window(Home)。财产（Episodes.Count），[COLOR=main_fg_90][B]，[/B][/COLOR]] episodes$COMMA $INFO[Window(Home)。Property(MusicVideos.Count),[COLOR=main_fg_90][B],[/B][/COLOR]]音乐视频$COMMA $INFO[Window(Home)。Property(Music.ArtistsCount),[COLOR=main_fg_90][B],[/B][/COLOR]] artists$COMMA $INFO[Window(Home).Property(Music.AlbumsCount),[COLOR=main_fg_90][B],[/B][/COLOR]]专辑和$INFO[Window(Home)。Property(Music.SongsCount),[COLOR=main_fg_90][B],[/B][/COLOR]]您资料库中的歌曲。"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Add here.

File: language/resource.language.zh_cn/strings.po [hunk 4]
```diff
@@ -0,0 +1,1604 @@
...
+msgstr "您总共观看了$INFO[Window(Home)。Property(TMDbHelper.TraktStats.Total.Minutes_d),[COLOR=main_fg_90][B],[/B][/COLOR]] days$COMMA $INFO[Window(Home).Property(TMDbHelper.TraktStats.Total.Minutes_h),[COLOR=main_fg_90][B],[/B][/COLOR]] hrs and $INFO[Window(Home).财产(TMDbHelper.TraktStats.Total.Minutes_mm),[COLOR=main_fg_90][B],[/B][/COLOR]]分钟内容。$INFO[窗口（主页）。财产(TMDbHelper.TraktStats.Recent.0.类型)] $INFO[窗口（主页）。财产(TMDbHelper.TraktStats.Recent.0.Episode.Title），[COLOR=main_fg_90][B]，[/B][/COLOR]]$INFO[窗口（主页）。财产(TMDbHelper.TraktStats.Recent.0.Show.Title），来自节目[COLOR=main_fg_90][B]，[/B][/COLOR]]$INFO[Window(Home)。财产(TMDbHelper.TraktStats.Recent.0.Movie.Title),[COLOR=main_fg_90][B],[/B][/COLOR]]$INFO[Window(Home).财产(TMDbHelper.TraktStats.Recent.0.Movie.Year），在[COLOR=main_fg_90][B]，[/B][/COLOR]]上映，最近观看了。"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Add here.

File: language/resource.language.zh_cn/strings.po [hunk 5]
```diff
@@ -0,0 +1,1604 @@
...
+msgstr "您已经观看了$INFO[Window(Home)。Property(TMDbHelper.TraktStats.Movies.Watched),[COLOR=main_fg_90][B],[/B][/COLOR]]电影和$INFO[Window(Home).Property(TMDbHelper.TraktStats.Shows.Watched),[COLOR=main_fg_90][B],[/B][/COLOR]] shows.$INFO[窗口（主页）。财产(TMDbHelper.TraktStats.Recent.0.类型)] $INFO[窗口（主页）。财产(TMDbHelper.TraktStats.Recent.0.Episode.Title），[COLOR=main_fg_90][B]，[/B][/COLOR]]$INFO[窗口（主页）。财产(TMDbHelper.TraktStats.Recent.0.Show.Title），来自节目[COLOR=main_fg_90][B]，[/B][/COLOR]]$INFO[Window(Home)。财产(TMDbHelper.TraktStats.Recent.0.Movie.Title),[COLOR=main_fg_90][B],[/B][/COLOR]]$INFO[Window(Home).财产(TMDbHelper.TraktStats.Recent.0.Movie.Year），在[COLOR=main_fg_90][B]，[/B][/COLOR]]上映，最近观看了。"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Add here.

File: language/resource.language.zh_cn/strings.po [hunk 6]
```diff
@@ -0,0 +1,1604 @@
...
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31519"
+msgid "A single reusable group of widgets"
+msgstr "一组可重复使用的小部件"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31520"
+msgid "Widgets for $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] icon on home screen"
+msgstr "$INFO[Container(22001)的小部件。主页屏幕上的ListItem.Label，[LOWERCASE]，[/LOWERCASE]]图标"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Add here.

File: language/resource.language.zh_cn/strings.po [hunk 7]
```diff
@@ -0,0 +1,1604 @@
...
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31521"
+msgid "Edit global widgets"
+msgstr "编辑全局小部件"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31522"
+msgid "Widgets for $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] category"
+msgstr "$INFO[Container(22001)的小部件。ListItem.Label，[LOWERCASE]，[/LOWERCASE]]类别"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Add Here.

File: language/resource.language.zh_cn/strings.po [hunk 8]
```diff
@@ -0,0 +1,1604 @@
...
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31524"
+msgid "Edit $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] submenu"
+msgstr "编辑 $INFO[Container(22001)。ListItem.Label，[LOWERCASE]，[/LOWERCASE]]子菜单"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31525"
+msgid "Edit $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] spotlight"
+msgstr "编辑 $INFO[Container(22001)。ListItem.Label，[LOWERCASE]，[/LOWERCASE]]聚光灯"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Add Here

File: language/resource.language.zh_cn/strings.po [hunk 9]
```diff
@@ -0,0 +1,1604 @@
...
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31523"
+msgid "Edit $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] widgets"
+msgstr "编辑 $INFO[Container(22001)。ListItem.Label，[LOWERCASE]，[/LOWERCASE]]小部件"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31524"
+msgid "Edit $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] submenu"
+msgstr "编辑 $INFO[Container(22001)。ListItem.Label，[LOWERCASE]，[/LOWERCASE]]子菜单"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Add Here.

File: language/resource.language.zh_cn/strings.po [hunk 10]
```diff
@@ -0,0 +1,1604 @@
...
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31522"
+msgid "Widgets for $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] category"
+msgstr "$INFO[Container(22001)的小部件。ListItem.Label，[LOWERCASE]，[/LOWERCASE]]类别"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31523"
+msgid "Edit $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] widgets"
+msgstr "编辑 $INFO[Container(22001)。ListItem.Label，[LOWERCASE]，[/LOWERCASE]]小部件"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Add Here.

File: language/resource.language.zh_cn/strings.po [hunk 11]
```diff
@@ -0,0 +1,1604 @@
...
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31526"
+msgid "Edit hub spotlight"
+msgstr "编辑 hub 聚光灯"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31527"
+msgid "Spotlight inside the $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] hub"
+msgstr "$INFO[Container(22001)内的聚光灯。ListItem.Label，[LOWERCASE]，[/LOWERCASE]] hub"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Here

File: language/resource.language.zh_cn/strings.po [hunk 12]
```diff
@@ -0,0 +1,1604 @@
...
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31528"
+msgid "Edit hub categories"
+msgstr "编辑 hub 类别"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31529"
+msgid "Categories inside the $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] hub"
+msgstr "$INFO[Container(22001)内的类别。ListItem.Label，[LOWERCASE]，[/LOWERCASE]] hub"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Here.

File: language/resource.language.zh_cn/strings.po [hunk 13]
```diff
@@ -0,0 +1,1604 @@
...
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31530"
+msgid "Edit hub widgets"
+msgstr "编辑 hub 小部件"
+
+#: /1080i/Custom_1115_Dialog_Shortcuts.xml
+msgctxt "#31531"
+msgid "Widgets inside the $INFO[Container(22001).ListItem.Label,[LOWERCASE],[/LOWERCASE]] hub"
+msgstr "$INFO[Container(22001)中的小部件。ListItem.Label，[LOWERCASE]，[/LOWERCASE]] hub"
```
Comment (jurialmunkey) [inline language/resource.language.zh_cn/strings.po]: Here.

**Review by jurialmunkey — CHANGES_REQUESTED** [review]
Thanks for submitting this. There are few issues where `$INFO[]` tags that shouldn't be translated have been translated. These will break in the skin if you change them.

Anything inside square brackets `[ ]` should not be translated.

**Comment by kalifun** [comment]
> Thanks for submitting this. There are few issues where `$INFO[]` tags that shouldn't be translated have been translated. These will break in the skin if you change them.
> 
> Anything inside square brackets `[ ]` should not be translated.

I'm very sorry, I didn't notice. I will revise it before submitting it.

**Comment by kalifun** [comment]
I have tried to modify it. Please help me see if there is any problem. @jurialmunkey

**Review by icarok99 — APPROVED** [review]

**Pull request merged**